src/index.html

```diff
@@ -1,13 +1,50 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        alt="mac"
+        class="card__img">
+      <div class="content">
```

Comment:
You already have block - `card`. It's not correct according BEM, check BEM rules - https://mate-academy.github.io/fe-program/css/typical-bem-mistakes
```suggestion
      <div class="content">
```

Reply:
I don`t understand suggested change

Reply:
![image](https://user-images.githubusercontent.com/82914630/167428907-2a586bcb-973a-4a2a-9bbc-596cd3c3e8e5.png)